src/CRISP.BackendChallenge/Repository/ContextRepository.cs

```diff
@@ -20,36 +21,97 @@ public IQueryable<T> Query()
     /// <inheritdoc />
     public IEnumerable<T> GetAll()
     {
-        throw new NotImplementedException();
+        var entities = _context.Employees.Include(e => e.Logins).ToList();
+        if (entities == null!)
+        {
+            return null!;
+        }
+
+        return (entities as IEnumerable<T>)!;
```

Comment:
The null forgiving operator (`!`) is meant to be used for situations where the compiler _thinks_ something can be null (and thus generates a warning), but the developer knows that functionally it can NEVER be null.

In this case, this could absolutely return null. The compiler is giving you a legitimate warning, there is no sense suppressing it.